### PR TITLE
Run precommit --all-files

### LINKED ISSUE #6283: Execute pre-commit run --all-files
Wait for https://github.com/mozilla/treeherder/issues/6190
and for https://github.com/mozilla/treeherder/pull/6277 to land.

The 2nd PR will actually fix the isort hook that causes a lot of files to be changed.

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
Fixes #6283 
Added a configuration for flake8 so that it does not fight with black
Add pre-commit -run--all-files to .travis.yml
Made necessary changes according to black

### CODE REVIEW

**Comment by armenzg** [comment]
Hi @SuyashSalampuria; thanks for this PR.
Could you please rebase over master?
We just merged #6332.

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[flake8]
```
Comment (armenzg) [inline .flake8:1]: Any changes for `flake8` configuration should be made inside of `setup.cfg`.

**Comment by SuyashSalampuria** [comment]
Thanks for it
That PR eased my work a lot
I will just do it

File: .travis.yml [hunk 2]
```diff
@@ -50,6 +50,7 @@ jobs:
         - pip install -r requirements.txt
         - yarn install
       script:
+        - pre-commit run --all-files
```
Comment (armenzg) [inline .travis.yml:53]: Could you please instead modify `tox.ini`?

**Pull request closed without merging**